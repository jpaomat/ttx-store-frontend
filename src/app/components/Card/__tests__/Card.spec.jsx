const Renderer = require('react-test-renderer/shallow');
const { Card } = require('../Card');

const renderer = new Renderer();

describe('Techtronix component', () => {
	it('should render component', () => {
		const component = renderer.render(<Card />);
		expect(component).toMatchSnapshot();
	});
});