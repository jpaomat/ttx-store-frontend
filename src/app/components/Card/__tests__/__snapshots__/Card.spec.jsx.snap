// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Techtronix component should render component 1`] = `
<div
  className="card"
>
  <img
    className="card__image"
    src="http:image-hp-test.com"
  />
  <div
    className="card__content"
  >
    <p
      className="card__content--brand"
    >
      Hp
    </p>
    <p
      className="card__content--model"
    >
      test model
    </p>
    <p
      className="card__content--price"
    >
      110
       â‚¬
    </p>
  </div>
  <Button
    disabledBtn={false}
    onClick={[Function]}
    size="sm"
  >
    <React.Fragment>
      <span>
        Agregar al carrito
      </span>
      <BsCart4
        className="cart-icon"
        size={14}
      />
    </React.Fragment>
  </Button>
</div>
`;
